be1-go/message/messagedata/chirp_broadcast.go

```diff
@@ -0,0 +1,9 @@
+package messagedata
+
+type ChirpBroadcast struct {
```

Comment:
Not formatted and not commented. Please look at the code around. This is the case for every other messages in this package!